orb/CronJob.yml

```diff
@@ -1,4 +1,4 @@
-apiVersion: batch/v1beta1
+apiVersion: batch/v1
```

Comment:
Love this.

Reply:
https://v1-20.docs.kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/
in 1.20 we only have v1beta1